AK/Span.h

```diff
@@ -105,10 +105,7 @@ class Span : public Detail::Span<T> {
 public:
     using Detail::Span<T>::Span;
 
-    ALWAYS_INLINE constexpr Span(std::nullptr_t)
```

Comment:
I think `nullptr` is much more readable than `{}`. I'd keep this.

Reply:
This constructor should at least be `explicit` since it takes only a single argument. However, it is unnecessary to even provide it. It is redundant with the default constructor and lets callers do implicit conversions. Removing this adds type safety (only a little bit though). 

 https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c46-by-default-declare-single-argument-constructors-explicit